### ISSUE TITLE
failed to run because of javascript grammar error?

### ISSUE THREAD

**Issue by Cleopatro** [FGRibreau/gron#5]
when I run `curl -s https://jsonplaceholder.typicode.com/users | gron | fgrep "company.name"`, it got an error like this:
```
/usr/local/lib/node_modules/gron/bin/gron.js:8
let d = '';
^^^

SyntaxError: Block-scoped declarations (let, const, function, class) not yet supported outside strict mode
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:387:25)
    at Object.Module._extensions..js (module.js:422:10)
    at Module.load (module.js:357:32)
    at Function.Module._load (module.js:314:12)
    at Function.Module.runMain (module.js:447:10)
    at startup (node.js:142:18)
    at node.js:939:3
```

**Comment by kfarst** [comment]
I believe this is directly related to https://github.com/FGRibreau/gron/issues/6

**Comment by FGRibreau** [comment]
Indeed, fixed, see #6
